pkg/p2p/libp2p/libp2p.go

```diff
@@ -564,8 +591,15 @@ func (s *Service) Connect(ctx context.Context, addr ma.Multiaddr) (address *bzz.
 	s.protocolsmu.RUnlock()
 
 	s.metrics.CreatedConnectionCount.Inc()
-	s.logger.Debugf("successfully connected to peer %s (outbound)", i.BzzAddress.ShortString())
-	s.logger.Infof("successfully connected to peer %s (outbound)", i.BzzAddress.Overlay)
+
+	var lightMode string
+
+	if i.Light {
```

Comment:
you could pull this part into the BzzAddress implementation, in the same way that we have ShortString (L601), so adding another method there to add this conditional formatting could be nice

Reply:
is this even valuable? I needed it only for local development, should we keep it or better discard it?

Reply:
can't hurt tbh. it would also be nice to have metrics in regards to light node connect/disconnects. Having some indication in the logs wouldn't hurt :+1: